pynucastro/templates/starkiller-microphysics/actual_network.F90.template

```diff
@@ -54,8 +54,7 @@ module actual_network
   integer, parameter :: i_drate_dt    = 2
   integer, parameter :: i_scor        = 3
   integer, parameter :: i_dscor_dt    = 4
-  integer, parameter :: i_dqweak      = 5
-  integer, parameter :: i_epart       = 6
+  integer, parameter :: i_eneut       = 4
```

Comment:
why two 4s in a row?

Reply:
These networks use a 4-element array "reactvec" to get evaluated rates from two subroutines:

- `table_rates.tabular_evaluate`: fills `reactvec` for tabulated rates containing (rate, drate/dT, 1.0, eneut) indexed by (i_rate, i_drate_dt, i_scor, i_eneut)

- `reaclib_rates.reaclib_evaluate`: fills `reactvec` for Reaclib rates containing (rate, drate/dT, screening, dscreening/dT) indexed by (i_rate, i_drate_dt, i_scor, i_dscor_dt)

Then we take the entries in `reactvec` and fill entries in `rate_eval % unscreened_rates` in `actual_rhs_module.evaluate_rates` and finally compute screened rates. When we access elements of `reactvec` we use either `i_eneut` or `i_dscor_dt` depending on whether the values came from `tabular_evaluate` or `reaclib_evaluate`.

Now that I'm looking at this, because we calculate screening separately from `reaclib_evaluate` we can do this more efficiently by returning 2 reals from `reaclib_evaluate` and 3 reals from `tabular_evaluate` without filling a temporary 4-element array.